aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -529,11 +529,15 @@ SparseTensor dense_to_sparse(const Tensor& self, c10::optional<c10::Layout> layo
                " conversion does not use the specified blocksize ", blocksize.value(), ".");
     }
     if (self.layout() == *layout) {
-      return self;
+      if (kStrided == *layout) {
+        // Returning clone to avoid
+        // RuntimeError: result.storage().use_count() == 1 INTERNAL ASSERT FAILED
+        return self.clone();
+      } else {
+        return self.alias();
```

Comment:
alias() here is technically not ok as the signature of this function says the output is not a view of the input (that's why the Dense case fails).
This can lead to silent correctness issues with autograd if you care about it.

Reply:
This code bit is out-of-date. So, resolving the discussion.